src/stravalib/model.py

```diff
@@ -377,9 +414,6 @@ class LatLon(LatLng, BackwardCompatibilityMixin, DeprecatedSerializableMixin):
     Enables backward compatibility for legacy namedtuple
```

Comment:
I believe this comment might have been from me when i was typing things. so i'm removing it.